source/reference/minio-mc/mc-license-register.rst

```diff
@@ -9,13 +9,18 @@
    :depth: 1
 
 .. mc:: mc support register
+.. mc:: mc license register
+
+.. note::
+
+   ``mc license register`` replaces the ``mc support register`` command.
```

Comment:
If we know what version in mc this happens, should state:

.. versionchanged:: RELEASE.<whatever>

  mc license register replaces mc support register

Reply:
This happened in `RELEASE.2022-07-15T09-20-55Z`

Reply:
👍
And thanks for saving me the research, @anjalshireesh!